### PR TITLE
Deprecating Get-AzSqlDatabaseSecureConnectionPolicy cmdlet

### PR DESCRIPTION
**Pull request by ayeletshpigelman**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Sql/Sql/ChangeLog.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Deprecating Get-AzSqlDatabaseSecureConnectionPolicy cmdlet
```
Comment (yaakoviyun) [inline src/Sql/Sql/ChangeLog.md]: what about the set cmdlet?

File: src/Sql/Sql/Secure Connection/Cmdlet/GetAzureSqlDatabaseSecureConnectionPolicy.cs [hunk 2]
```diff
@@ -22,6 +22,7 @@ namespace Microsoft.Azure.Commands.Sql.SecureConnection.Cmdlet
     /// Returns the secure connection policy of a specific database.
     /// </summary>
     [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlDatabaseSecureConnectionPolicy", SupportsShouldProcess = true), OutputType(typeof(DatabaseSecureConnectionPolicyModel))]
+    [Obsolete("Note that secure connection is deprecated and this command will be removed in a future release. Please use the SQL database blade in the Azure portal to view the connection strings", false)]
```
Comment (yaakoviyun) [inline src/Sql/Sql/Secure Connection/Cmdlet/GetAzureSqlDatabaseSecureConnectionPolicy.cs]: close with PM.
I will point PS customers to the portal

File: src/Sql/Sql/help/Az.Sql.md [hunk 3]
```diff
@@ -117,7 +117,8 @@ Gets the geo-replication links between an Azure SQL Database and a resource grou
 Retrieves the distinct restore points from which a SQL Data Warehouse can be restored.
 
 ### [Get-AzSqlDatabaseSecureConnectionPolicy](Get-AzSqlDatabaseSecureConnectionPolicy.md)
-Gets the secure connection policy for a database.
+Gets the secure connection policy for a database.
```
Comment (yaakoviyun) [inline src/Sql/Sql/help/Az.Sql.md]: do we have cmdlet for server connection policy too?

File: src/Sql/Sql/Secure Connection/Cmdlet/GetAzureSqlDatabaseSecureConnectionPolicy.cs (hunk 2 above)
Comment (MiYanni) [reply src/Sql/Sql/Secure Connection/Cmdlet/GetAzureSqlDatabaseSecureConnectionPolicy.cs]: Do not use `Obsolete`. Use our breaking change attributes: https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md

File: src/Sql/Sql/help/Az.Sql.md (hunk 3 above)
Comment (ayeletshpigelman) [reply src/Sql/Sql/help/Az.Sql.md]: no

File: src/Sql/Sql/Secure Connection/Cmdlet/GetAzureSqlDatabaseSecureConnectionPolicy.cs (hunk 2 above)
Comment (ayeletshpigelman) [reply src/Sql/Sql/Secure Connection/Cmdlet/GetAzureSqlDatabaseSecureConnectionPolicy.cs]: done

**Pull request merged**